samples/cpp/tutorial_code/objectDetection/objectDetection.cpp

```diff
@@ -13,8 +13,8 @@ using namespace cv;
 void detectAndDisplay( Mat frame );
 
 /** Global variables */
-String face_cascade_name = "haarcascade_frontalface_alt.xml";
-String eyes_cascade_name = "haarcascade_eye_tree_eyeglasses.xml";
+String face_cascade_name = "../../data/haarcascades/haarcascade_frontalface_alt.xml";
+String eyes_cascade_name = "../../data/haarcascades/haarcascade_eye_tree_eyeglasses.xml";
```

Comment:
Is this change really helpful?
I think that ability to read these paths from the command line would be useful here.